src/Data/List/Relation/Binary/Equality/Setoid.agda

```diff
@@ -121,7 +121,12 @@ module _ {P : Pred A p} (P? : U.Decidable P) (resp : P Respects _≈_)
   filter⁺ xs≋ys = PW.filter⁺ P? P? resp (resp ∘ sym) xs≋ys
 
 ------------------------------------------------------------------------
--- filter
+-- reverse
+
+infixr 5 _ʳ++⁺_
+
+_ʳ++⁺_ : ∀{xs xs′ ys ys′} → xs ≋ xs′ → ys ≋ ys′ → xs ʳ++ ys ≋ xs′ ʳ++ ys′
```

Comment:
The corresponding proof for `++` doesn't use infix notation so I'm not sure this one should either. If we use copy `++` in using `ws xs ys zs` as variables as well then it also fits within the line length!

Reply:
Isn't `++` off here?  I'd expect that if we have an infix operator like `++`, then its predicate/relation lifting is also an infix operator.  Examples of this is `Data.List.All._::_` which is the predicate lifting of `Data.List._::_`.

Reply:
> Isn't ++ off here?

Hmm well if it is off, it's off everywhere else in the library: 
- [Any](https://github.com/agda/agda-stdlib/blob/c82ed45d6b6b1322b2b1f55c52edd1a38ab56926/src/Data/List/Relation/Unary/Any/Properties.agda#L331)
- [All](https://github.com/agda/agda-stdlib/blob/c82ed45d6b6b1322b2b1f55c52edd1a38ab56926/src/Data/List/Relation/Unary/All/Properties.agda#L374)
- [Unique](https://github.com/agda/agda-stdlib/blob/c82ed45d6b6b1322b2b1f55c52edd1a38ab56926/src/Data/List/Relation/Unary/Unique/Setoid/Properties.agda#L46)
- [Permutation](https://github.com/agda/agda-stdlib/blob/c82ed45d6b6b1322b2b1f55c52edd1a38ab56926/src/Data/List/Relation/Binary/Permutation/Propositional/Properties.agda#L86)
- [Prefix](https://github.com/agda/agda-stdlib/blob/c82ed45d6b6b1322b2b1f55c52edd1a38ab56926/src/Data/List/Relation/Binary/Prefix/Heterogeneous/Properties.agda#L72)
- [Pointwise](https://github.com/agda/agda-stdlib/blob/c82ed45d6b6b1322b2b1f55c52edd1a38ab56926/src/Data/Vec/Relation/Binary/Pointwise/Inductive.agda#L159) (for Vec)
- etc. (at least 5 more places)

I'd suggest if you do want to change it, then the best course of action is to make this PR consistent with the current form, and then open a new PR which changes it **everywhere** in the library.

Reply:
Seems fair for the sake of conservativity.

I guess this decision (infix or not) is also a topic for the style guide...